control-plane-operator/controllers/hostedcontrolplane/v2/etcd/etcd-init.sh

```
@@ -7,11 +7,17 @@ RESTORE_URL=${!RESTORE_URL_VAR}
 
 # When downloading from S3, curl can succeed even if the object doesn't exist
 # and also when a pre-signed URL is expired.
-# In this case we get an XML file which can be detected with `file` so we show
-# the contents via the logs then exit with an error status
+# In this case we get an XML error response which we detect and show via the logs
+# then exit with an error status.
 curl -o /tmp/snapshot ${RESTORE_URL}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)